src/pages/Home.jsx

```diff
@@ -13,26 +20,85 @@ import Paper from '@mui/material/Paper';
  * @name Home
  * @returns {React.ReactElement} The home page
  */
-const Home = () => (
-  <Container component="main" maxWidth="s">
-    <Box
-      sx={{
-        marginTop: 18,
-        marginBottom: 18,
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        textAlign: 'center',
-        padding: '20px'
-      }}
-    >
-      <Paper elevation={2} sx={{ display: 'inline-block', mx: '2px', padding: '20px' }}>
+const Home = () => {
+  const theme = useTheme();
+  const isReallySmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+  return (
+    <Container component="main">
+      <Box
+        sx={{
+          margin: '18px 0',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          textAlign: 'center',
+          padding: '20px'
+        }}
+      >
         <section id="home">
-          <h1>HOME PAGE</h1>
+          <Stack
+            width={isReallySmallScreen ? 1 : 3 / 3}
+            alignItems="center"
+            justifyContent="center"
+          >
+            <ReusableHomeSection
+              src="./src/assets/web-security.png"
+              alt="Web security"
+              title="Keep Your Documents Safe and Secure Using Decentralized Technology"
+              description="Our innovatice solution empowers individuals to manage their critical documents and control access for trusted organizations. PASS simplifies service access, enabling seamless documents requests and secure data sharing for a smoother support process. "
+              button="Request a Demo"
+              href="mailto:hugh@codeforpdx.org"
+              label="Demo request button"
+            />
+            <ReusableHomeSection
+              src="./src/assets/app.png"
+              alt="Web App"
+              title="An App Built for Case Workers"
+              description="Pass allows users to quickly and securely share documents of their clients within the app. The app helps case workes verify and share documents such as ID and proof of income while having total control of the documents."
+            />
+            <ReusableHomeSection
+              src="./src/assets/key-features.png"
+              alt="Key Features"
+              title="Key Features"
+            />
+            <ReusableKeyFeatures
```

Comment:
since all these icons that are passed to `ReusableKeyFeatures` are `color: 'Purple.main'`
**&**
in `ReusableKeyFeatures` the `Typography`, located in the same `Box` as this icon, is also the same color

Then, we should just set the `Box` in `ReusableKeyFeatures` with `color: 'Purple.main'` and so we don't have to do this each time here (or on the `Typography` within the box either.)